combat-trainer.lic

```diff
@@ -1314,6 +1314,11 @@ class AttackProcess
     pause
     waitrt?
 
+    if reget(20, 'increase the accuracy')
```

Comment:
This is tricky. Just finishing the array doesn't mean that it was successful, sometimes it inputs actions that get triggers are completed, but don't really. This happens when something dies and you gotta switch targets, it leaves, it's out of range, etc. I thought about a flag that checked if doing the combo, but you still need to check if that text was received to know you have done it. A way to make this work without it, is just to remove the reget and let it attempt to go through, if it finds that it's been done and the combo is on cooldown, it sets the flag to false. This is actually part of the backup checks to make sure it doesn't continuously loop. Removing reget will put this as the main check. The only issue is that it might appear spammy the first time it goes through to find out if it's on cooldown.